test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -146,6 +147,81 @@ public HubConnectionTests()
 
         [Theory]
         [MemberData(nameof(HubProtocols))]
+        public async Task DynamicHubFixedMessage(IHubProtocol protocol)
```

Comment:
This is a copy of the other test - why not make the test hub look alike and then have the same test that tests regular and dynamic hubs? From the client perspective they should behave the same way.